x/group/client/testutil/tx.go

```diff
@@ -1122,6 +1123,13 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupPolicyMetadata() {
 	}
 }
 
+// TestTxCreateProposal tests submitting proposal.
+//
+// Please don't rename this to TestTxSubmitProposal. It will redefine the order
+// of the tests being run in this file with `go test`, and will mess up the
+// proposal ids in other tests (e.g. voting or Exec tests).
+// This is a headache, but requires a bigger refactor of all tests in this file
+// so that each one is independent.
```

Comment:
can we create an issue for this then? and can be refactored in upcoming versions.